core/src/main/java/org/infinispan/dataconversion/PersistenceMarshallerEncoder.java

```diff
@@ -0,0 +1,26 @@
+package org.infinispan.dataconversion;
+
+import org.infinispan.commons.dataconversion.EncoderIds;
+import org.infinispan.commons.dataconversion.MarshallerEncoder;
+import org.infinispan.commons.dataconversion.MediaType;
+import org.infinispan.marshall.persistence.PersistenceMarshaller;
+
+/**
+ * @since 10.0
+ */
+public class PersistenceMarshallerEncoder extends MarshallerEncoder {
```

Comment:
Why is this needed? Generally speaking, persistence and encoding/transcoding live in different layers: while transcoding is between the user and the cache API (it does not happen throughout the interceptors), persistence is between the cache and the stores.

Reply:
Who would use this? I don't see this in use other than registering it in the encoder registry.